Frontend/tailored-brand-poc/src/app/all-invoices/all-invoices.component.html

```diff
@@ -64,31 +64,32 @@
         <button type="button" class="btn btn-dark" (click)="paginateInvoices()">Search</button>
       </div>
     </div>
-    <hr>
     <label>Invoices:</label>
     <div class="table-responsive">
       <table class="table table-striped">
         <thead class="thead-dark">
-          <th class="right">SNo.</th>
-          <th class="right">Invoice No.</th>
-          <th class="right">Invoice Date</th>
-          <th class="right">Order No.</th>
-          <th>Invoice Amount</th>
+          <th width="10%">SNo.</th>
+          <th width="25%" class="right">Invoice No.</th>
```

Comment:
Instead of using the custom class `right`, make use of the equivalent bootstrap class `text-right`.